config/controllers_deployment.yml

```diff
@@ -43,11 +43,11 @@ spec:
           value: #@ data.values.workloads_namespace
         resources:
           limits:
-            cpu: 100m
-            memory: 200Mi
+            cpu: 1000m
```

Comment:
This controller is responsible for staging (it watches kpack build/images) and some route reconciliation. We should definitely let it use _at least_ a full CPU if not more. I think this must have been a typo for it to be so low.